js/modules/Monaco/MonacoEditor.js

```diff
@@ -166,6 +169,62 @@ export default class MonacoEditor {
             });
         }
     }
+
+    getTwigCompletions() {
+        const keywords = [
+            // (opening) tags
+            'apply', 'autoescape', 'block', 'for', 'if', 'macro', 'set',
+            // closing tags
+            'endapply', 'endautoescape', 'endblock', 'endfor', 'endif', 'endmacro', 'endset',
+            // literals
+            'true', 'false', 'null'
+        ];
+        const operators = [
+            'in', 'is', 'and', 'or', 'not', 'b-and', 'b-xor', 'b-or', 'matches', 'starts with', 'ends with', 'has some', 'has every',
+        ];
+        const functions = ['min', 'max', 'random', 'range'];
```

Comment:
It is available both in filters and functions.
```suggestion
        const functions = ['date', 'min', 'max', 'random', 'range'];
```